index.php

```diff
@@ -5,21 +5,38 @@
     <title>Netflix Series Collection</title>
 </head>
 <body>
+<h2>Complete all fields to add a new series to your collection</h2>
+<!-- -->
+<form method="post" action="adddata.php">
+    Series Title:
+    <input type="text" name="seriestitle">
```

Comment:
Can use the `required` attribute for these input fields (step 1 of the 4 that we talked about)

Reply:
required attribute added to all fields